src/lib/middleware/errorHandler.ts

```diff
@@ -47,3 +48,23 @@ export function errorHandlerMiddleware(
 
   res.status(code).render(file, { code, detail, message })
 }
+
+export function setupErrorHandling(app) {
+  app.get("*", (req, res, next) => {
+    const err = new Error()
+    // @ts-ignore -- FIXME: status does not exist on err
+    err.status = 404
+    err.message = "Not Found"
+    next(err)
+  })
+
+  if (SENTRY_PRIVATE_DSN) {
+    // Old Sentry SDK.
+    app.use(RavenServer.errorHandler())
+
+    RavenServer.config(SENTRY_PRIVATE_DSN).install()
+    app.use(RavenServer.requestHandler())
```

Comment:
I have a feeling that this needs to stay as one of the first middleware added to properly hook into force.

Reply:
Let me look at the docs, I remember a while back it said it had to go last, but let me see.

Reply:
Ah, yup: https://docs.sentry.io/clients/node/integrations/#express

Will fix